pandas/_libs/lib.pyx

```diff
@@ -107,24 +109,57 @@ def memory_usage_of_objects(arr: object[:]) -> int64_t:
 def is_scalar(val: object) -> bint:
     """
     Return True if given value is scalar.
-
+    
+    Parameters
+    ----------
+    val : object
     This includes:
-    - numpy array scalar (e.g. np.int64)
-    - Python builtin numerics
-    - Python builtin byte arrays and strings
-    - None
-    - instances of datetime.datetime
-    - instances of datetime.timedelta
-    - Period
-    - instances of decimal.Decimal
-    - Interval
-    - DateOffset
+    * numpy array scalar (e.g. np.int64)
+    * Python builtin numerics
+    * Python builtin byte arrays and strings
+    * None
+    * instances of datetime.datetime
+    * instances of datetime.timedelta
+    * Period
+    * instances of decimal.Decimal
```

Comment:
can you remove all the `instances of` here. All the items in the list what you can check are the instances